holoviews/tests/util/test_utils.py

```diff
@@ -26,6 +26,8 @@ class TestOutputUtil(ComparisonTestCase):
     def setUp(self):
         if notebook is None:
             raise SkipTest("Jupyter Notebook not available")
+        from holoviews.ipython import notebook_extension
```

Comment:
It seems like IPython was not part of the `sys.module` at the import time - i'm not entirely sure why.